cmd/admin-handlers-site-replication.go

```diff
@@ -398,6 +410,13 @@ func (a adminAPIHandlers) SiteReplicationEdit(w http.ResponseWriter, r *http.Req
 	writeSuccessResponseJSON(w, body)
 }
 
+func getSREditOptions(r *http.Request) (opts madmin.SREditOptions) {
+	q := r.Form
+	opts.DisableILMExpiryReplication = q.Get("disableILMExpiryReplication") == "true"
+	opts.EnableILMExpiryReplication = q.Get("enableILMExpiryReplication") == "true"
+	return
```

Comment:
```suggestion
	opts.DisableILMExpiryReplication = r.Form.Get("disableILMExpiryReplication") == "true"
	opts.EnableILMExpiryReplication = r.Form.Get("enableILMExpiryReplication") == "true"
	return
```

Reply:
Sent a new PR https://github.com/minio/minio/pull/18499 for the same